from django.test import TestCase
from django.core.exceptions import ValidationError


class MixinsTestCase(TestCase):
    pass
